tests/integration/tools/test_google_docs.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+import os
+from tests.utils.structure_runner import run_structure, OUTPUT_RULESET, PROMPT_DRIVERS, prompt_driver_id_fn
```

Comment:
Change `PROMPT_DRIVERS` to `TOOKIT_TASK_CAPABLE_PROMPT_DRIVERS`

Reply:
Done